packages/adblocker/src/filters/network.ts

```diff
@@ -1265,6 +1276,14 @@ export default class NetworkFilter implements IFilter {
     return this.optionValue;
   }
 
+  public get removeparam(): string | undefined {
+    if (!this.isRemoveParam()) {
+      return undefined;
+    }
+
+    return this.optionValue || '';
```

Comment:
We should probably never return an empty string here.

Reply:
Correct, I checked that there's no chance of `optionValue` to be undefined.